core/templates/pages/about-page/about-page.component.html

```diff
@@ -246,4 +246,12 @@ <h3 [innerHTML]="'I18N_ABOUT_PAGE_SPEECH_BUBBLE' | translate"></h3>
   .oppia-about-page-extra-info {
     height: 245px;
   }
+  .oppia-about-three-cols-tab-create-text
```

Comment:
The naming is better now, but you don't need three separate classes for this. You can use .oppia-about-page-about-tab-image-caption-text (or possibly just .oppia-about-page-image-caption-text) -- this is an example of a name that describes the three things you're handling here, and nothing else. The point of CSS is to apply the same style to similar elements regardless of the specific text that each element contains.

(I'm also not sure what you mean by "three cols tab" in the class name you've chosen.)